### PR TITLE
Misc queue from 2018 (sorry, lost sources)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -11401,6 +11404,7 @@ puchasing->purchasing
 Pucini->Puccini
 Puertorrican->Puerto Rican
 Puertorricans->Puerto Ricans
+puffer->buffer
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Remove this, it's a perfectly valid word.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11681,6 +11685,7 @@ reconaissance->reconnaissance
 reconcilation->reconciliation
 reconfiged->reconfigured
 recongized->recognized
+recongnizes->recognizes
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the ses variant too please.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -11915,6 +11922,7 @@ reguster->register
 rehersal->rehearsal
 rehersing->rehearsing
 reicarnation->reincarnation
+reification->ratification
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: reunification should probably be a suggestion too, and perhaps more likely from what's been typed.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -11937,6 +11945,7 @@ reivison->revision
 reknown->renown
 reknowned->renowned
 rekursion->recursion
+rekursive->recursive
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add rekursed->recursed too

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Actually, it's a valid word: https://www.merriam-webster.com/dictionary/reification

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Well spotted @luzpaz

**Comment by peternewman** [comment]
Can you fix the comments here and the conflicts @Gelma ?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by Gelma** [comment]
> Can you fix the comments here and the conflicts @Gelma ?

I'm going to push

**Pull request merged**